zerver/lib/actions.py

```diff
@@ -1172,6 +1172,17 @@ def get_subscriber_emails(stream, requesting_user=None):
     subscriptions = subscriptions_query.values('user_profile__email')
     return [subscription['user_profile__email'] for subscription in subscriptions]
 
+def subscriber_ids_using_stream_dict(user_profile, stream_dict):
+    # type: Iterable[Mapping[str, Any]] -> List[int]
```

Comment:
@krtkmj Now that you've updated your diff to use `subscriber_map`, this function is no longer being used and should be removed.